### PR TITLE
Fixes fugitive hunters not getting team huds

### LINKED ISSUE #64410: Fugitive Hunters don't have their team antag hud
## Round ID:
[177236](https://scrubby.melonmesa.com/round/177236)

## Reproduction:

- Join fugitive hunter team
- there is no team hud to identify your team mates like in the past

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Fugitive hunters get a hud to find their partners

## Why It's Good For The Game
closes #64410 
helps the hunters work as a team, quickly spotting them even in a crowd

## Changelog

:cl:
fix: fugitive hunters get team hud
/:cl:

### CODE REVIEW

File: code/modules/antagonists/fugitive/hunter.dm [hunk 1]
```diff
@@ -55,6 +55,15 @@
 /datum/antagonist/fugitive_hunter/get_team()
 	return hunter_team
 
+/datum/antagonist/fugitive_hunter/apply_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	handle_clown_mutation(M, mob_override ? null : "Your training has allowed you to overcome your clownish nature, allowing you to wield weapons without harming yourself.")
+	add_team_hud(M)
+
+/datum/antagonist/rev/remove_innate_effects(mob/living/mob_override)
```
Comment (JohnFulpWillard) [inline code/modules/antagonists/fugitive/hunter.dm:63]: this is still on rev antags

File: code/modules/antagonists/fugitive/hunter.dm [hunk 2]
```diff
@@ -55,6 +55,15 @@
 /datum/antagonist/fugitive_hunter/get_team()
 	return hunter_team
 
+/datum/antagonist/fugitive_hunter/apply_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	handle_clown_mutation(M, mob_override ? null : "Your training has allowed you to overcome your clownish nature, allowing you to wield weapons without harming yourself.")
```
Comment (JohnFulpWillard) [inline code/modules/antagonists/fugitive/hunter.dm:60]: i dont think this is necessary? This is a ghost role

**Comment by etherware-novice** [comment]
do i..mark as open now?

**Comment by JohnFulpWillard** [comment]
> do i..mark as open now?

Once you think you addressed all reviews, you can mark your PRs as ready to review, no need to ask us

File: code/modules/antagonists/fugitive/hunter.dm [hunk 3]
```diff
@@ -55,6 +55,9 @@
 /datum/antagonist/fugitive_hunter/get_team()
 	return hunter_team
 
+/datum/antagonist/fugitive_hunter/apply_innate_effects()
+	add_team_hud(owner.current)
```
Comment (JohnFulpWillard) [inline code/modules/antagonists/fugitive/hunter.dm:59]:
```suggestion
/datum/antagonist/fugitive_hunter/apply_innate_effect(mob/living/mob_override)
	add_team_hud(mob_override || owner.current)
```

There's some cases (on body transfer) that needs to take into account a mob override.

**Pull request merged**